paderbox/utils/random_utils.py

```diff
@@ -2,6 +2,18 @@
 from functools import wraps
 
 
+def str_to_random_state(string):
+    """
+    This functions outputs a consistent random state dependent on
+    an input string.
+    """
+    from hashlib import md5
+    hash_value = md5(string.encode())
```

Comment:
In https://github.com/fgnt/sms_wsj/blob/f0cc4402a7111cd603c617a21170b0e0ddb90d48/sms_wsj/database/create_rirs.py#L63 we use sha256, can you use that?
Also, `encode("utf-8")` should be more robust.